### DISCUSSION TITLE
Welcome to native_db Discussions!

### DISCUSSION THREAD

**Discussion by vincent-herlemont** [vincent-herlemont/native_db#67] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by q-b** [comment]
Hi, I'm just lurking for now... Good luck and thank you for native_db! 🚀

**Reply by vincent-herlemont** [reply]
@q-b Thank you :heart: !
